### PR TITLE
Added parallel Zoey

### PR DESCRIPTION
**Pull request by Chr1Z93**

### CODE REVIEW

File: src/arkhamdb/ArkhamDb.ttslua [hunk 1]
```diff
@@ -198,7 +198,7 @@ do
       local altArt     = { front = "normal", back = "normal" }
 
       -- translating front ID
-      if altFrontId > 90000 and altFrontId < 90050 then
```
Comment (argonui) [inline src/arkhamdb/ArkhamDb.ttslua:201]: This is a pretty serious nightmare of maintenance. Are there other ways to determine parallel or not?

**Review by argonui — DISMISSED** [review]

File: objects/AllPlayerCards.15bb07/ZoeySamarasParallel.98a0e2.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "posZ": -73.088,
+    "rotX": 0,
+    "rotY": 180,
+    "rotZ": 0,
+    "scaleX": 1.15,
+    "scaleY": 1,
+    "scaleZ": 1.15
+  },
+  "Value": 0,
+  "XmlUI": ""
```
[the remark this answers is not in the document]
Comment (Chr1Z93) [reply objects/AllPlayerCards.15bb07/ZoeySamarasParallel.98a0e2.json:61]: good catch, fixed

File: src/arkhamdb/ArkhamDb.ttslua (hunk 1 above)
Comment (Chr1Z93) [reply src/arkhamdb/ArkhamDb.ttslua:201]: Very valid concern and this was indeed a pain in the ass to implement, but unfortunately the ArkhamDB Api just provides a single altId. We have to use this to discern whether this means it's a parallel investigator, revised core alt art or promo art.

**Review by Entrox-Licher — APPROVED** [review]
Valid maintainability concerns raised, but as far as the new cards themselves, they look good to me

**Pull request merged**